### PR TITLE
DevKit updates for 2.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Comment by OskarStark** [comment]
@VincentLanglet can you please Check this PR?

File: README.md [hunk 1]
```diff
@@ -1,63 +1,43 @@
-Cache
```
Comment (OskarStark) [inline README.md:1]: The README change seem to be wrong 😑

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Comment by VincentLanglet** [comment]
@OskarStark I need https://github.com/sonata-project/dev-kit/pull/1342

**Comment by OskarStark** [comment]
Lets recreate it

**Pull request closed without merging**